### DISCUSSION TITLE
Automatic buy for Amazon

### DISCUSSION THREAD

**Discussion by DerHOgi** [jef/streetmerchant#1471] [Ideas and feature requests]
Looks like there are thousands of bots are running that not only check the status, if a product is on stock or not (in my case a PS5), no they must also buy the product directly. If I get the alert from the bot, that an item is on stock, it's allready sold out after I opened Amazon. Even if I'm sitting directly at the computer and only have to reload the website that is already open, no chance! This all must happen in seconds or milliseconds. This is really crazy stuff. How is that possible? Is there an API or something like that on Amazon that can be used to buy directly?

BTW: Bot checks Amazon every 5 seconds. No chance at all with longer intervals! I use a VPN service that automatic changes the IP every 100 requests or directly if a captcha is detected.

**Comment by jackevans211** [comment]
yes would be nice to be able to get someone with the skils to program that
